spec/game_spec.rb

```diff
@@ -65,4 +65,57 @@ def print_message(string)
       expect(console.print_message(console.printed_string)).to eq(expected_output)
     end
   end
+
+  describe 'game_over?' do
+    it 'returns true if there is a winning board' do
```

Comment:
Does a board win? Do you mean "returns true if the board has a winner"?

Reply:
What about a winning combination? I didn't write "a winner" because what is a winner? I figured winner implied a player. (nitpicky answer :upside_down_face:)

Reply:
It's up to you. I'm just trying to get you to think more critically about what your tests are communicating and how your code reads